src/avram/callbacks.cr

```diff
@@ -1,29 +1,77 @@
 module Avram::Callbacks
+  # :nodoc:
+  macro conditional_error_for_inline_callbacks(callback, method_name, condition)
+    \{%
```

Comment:
@jwoertink could you please elaborate on why the leading slash is needed here? I think this is causing the compiler to report the incorrect location.

As I can see from the doc it is used to prevent evaluating a macro **the outer macro**, which is not the case here.

https://crystal-lang.org/reference/syntax_and_semantics/macros.html#nested-macros

Reply:
I have the macro interpolation in this, and without the leading slashes, the compile-time error produces the curly brace literals. 

```
# with the leading slash, I get
You must pass a Symbol to `if`

# without the leading slash, I get
You must pass a Symbol to `{{ condition.id }}`
```